### ISSUE TITLE
Can this work with SSL?

### ISSUE THREAD

**Issue by jmesberg** [AYapejian/node-red-contrib-home-assistant#29]
I have 40+ automations created that are working GREAT with your set of nodes, but today I decided it was time to get more secure in my system so I set up duckdns and lets encrypt (successfully as far as home assistant goes). As I expected all the nodes that have a server config item immediately went to disconnected but fixing the server address in the config to my new duckdns address doesn't work....they are staying disconnected. What am i missing here?

**Comment by walthowd** [comment]
For SSL you need port forwarding and NAT loopback enabled on your router.

**Comment by jmesberg** [comment]
Oddly I got it to work simply by adding a fresh node with the server config pointing to the new SSL enabled duckdns address. As soon as I deployed it all the other nodes went online immediately. (To be clear: I of course had already tried to update the server address in the config of the existing nodes. That did not work. Only when I added a fresh node did they all ‘light up’). 

Thank you for all of your great work here! I’ve moved everyone of my automations from yaml to Node-RED due to your work. 


John
